_posts/2021-12-16-update-1-2-2.markdown

```diff
@@ -11,7 +11,11 @@ excerpt: "As indicated in the previous blog post, CVE-2021-45046 was issued shor
 
 ---
 
+### Update 2021-12-22:
 
+OpenSearch version 1.2.3 is now available. This release updates the version of Log4j used in OpenSearch to Log4j 2.17.0 as recommended by the advisory in [CVE-2021-45105](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45105). Additionally, this release addresses a [regression introduced in OpenSearch 1.2.0](https://github.com/opensearch-project/OpenSearch/issues/1734) for those using repository-azure plugin. All users of OpenSearch should update to OpenSearch 1.2.3. [Learn more](/blog/releases/2021/12/update-1-2-3/).
+
+### Update your clusters to 1.2.2
```

Comment:
Update to 1.2.3 and mention both CVEs.

Reply:
This is the old blog. Shortening text to be more clear.